test/testdata/packager/lsp_features/bart/bart.rb

```diff
@@ -3,7 +3,7 @@
 
 module Bart
   #    ^^^^ def: bartmod
-  #    ^^^^ symbol-search: "Bart", name = "Bart", container = "Bart"
```

Comment:
👀 What the heck did `container = "Bart"` mean before? was that just like a really awkward artifact of how we modeled packages?

Reply:
This took a lot of puzzling with @froydnj earlier today: we're pretty sure that it was using the mangled package name as the container for the name in the response, which is why we needed to change this to `(nothing)` when no longer rewriting the tree.